Musicplayer.css

```diff
@@ -8,62 +8,32 @@ body{
 
 .grid-container{
 	display: grid;
-	grid-template-columns: 15% 10% 75%;
-	grid-template-rows: auto auto auto auto auto;
+	grid-template-columns: 15% 35% 50%;
+	grid-template-rows: 58px auto auto auto auto;
 	height: 100vh;
 }
 
-.img_1{
```

Comment:
21/9: I remove this because I think it should be easier if I dealing with the nav first. But I will include banner and footer in the bottom.